### ISSUE TITLE
Output driven input won't work when transform results contain decimal places

### ISSUE THREAD

**Issue by JaimeLeon2** [MobiFlight/MobiFlight-Connector#1234]
**Describe the bug**
There seems to be an issue in MF 9.7.1.1 with output driven input when Transforms are used in the output config.  The input action code apparently has a problem when the result of the Transform has decimal places. Using Round() solved the issue.

**Steps To Reproduce**
1. Setup an output driven input config
2. Make a formula that results in a number with decimal places
3. Use $ in the Display tab, Input action to transfer the Transform result to the input
4. Note nothing is received in the Input Action.
5. If Round($,0) is used then it all works again.

**Expected behavior**
The transform result is transferred to the Input Action without problems

**Actual behavior**
Nothing apparently happens when there are decimal places in the number to be transferred.

**Details**
 - MobiFlight Version: 9.7.1.1
 - FlightSim: MSFS
 - AddOn: Irrelevant

**Additional context**
None

**Comment by DocMoebiuz** [comment]
yes, it expects an integer as value

<img width="658" alt="image" src="https://github.com/MobiFlight/MobiFlight-Connector/assets/86157512/6c20c603-c50d-4ab6-b5fd-1ee458b6fcd9">

if it can't parse it, it still is a 0. Which is Button Released.

**Comment by DocMoebiuz** [comment]
I assume that you wanted to use a AnalogInputEvent, right?

**Comment by JaimeLeon2** [comment]
> I assume that you wanted to use a AnalogInputEvent, right?

Yes, analog input. I think decimals on the code to the Sim side are accepted and in some cases necessary. Mobiflight does not currently allow it.

**Comment by DocMoebiuz** [comment]
True. This is then rather a feature request then a bug report. Nice :D
